cunumeric/module.py

```diff
@@ -2442,6 +2503,156 @@ def diag_indices_from(arr: ndarray) -> tuple[ndarray, ...]:
     return diag_indices(arr.shape[0], arr.ndim)
 
 
+def tril_indices(
+    n: int, k: int = 0, m: Optional[int] = None
+) -> tuple[ndarray, ...]:
+    """
+    Return the indices for the lower-triangle of an (n, m) array.
+
+    Parameters
+    ----------
+    n : int
+        The row dimension of the arrays for which the returned
+        indices will be valid.
+    k : int, optional
+        Diagonal offset (see `tril` for details).
+    m : int, optional
+        The column dimension of the arrays for which the returned
+        arrays will be valid.
```

Comment:
"the returned indices"

This typo is present in the original NumPy documentation too, but nobody says we shouldn't do better!